examples/Cargo.toml

```diff
@@ -0,0 +1,47 @@
+[package]
```

Comment:
I'm not sure that one package will be more convenient than multiple. Imagine that we would have complicated example when we need different dependencies versions or different cargo features, then one `Cargo.toml` is not been convenient for that cases.

Maybe I have not known something about this approach. If it is truth please let me know